### PR TITLE
Day7_Routing_Assignment

### PR DESCRIPTION
**Pull request by sow2245**
1. Structing project with folders.
2. Installed React Router.
3. Added Routing.
4. Created new component AboutUs.
5. Created new ErrorComponent and Implemented errorelement.
6. Using useRouteError showing the error message.
7. Using Outlet and children to Implement SPA[Single Page Application].
8. Using useParams to pass id as parameter while routing.
9. Created new TeamMemberComponent.
10. Adding Link Functionality.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -50,9 +57,31 @@ const BodyComponent=()=>{
...
+    {
+        path : "/aboutus",
+        element : <AboutUsComponent/>
+    }
+]);
+
 const root = ReactDom.createRoot(document.getElementById("root"));
-root.render(<AppLayout/>);
\ No newline at end of file
+root.render(<RouterProvider router={appRouter}/>);
```
Comment (vickydarlinn) [inline src/App.js:87]: Refactor app.js file it is looking very conjusted.

File: src/App.js [hunk 2]
```diff
@@ -50,9 +57,31 @@ const BodyComponent=()=>{
...
     </>
 );
 
+const appRouter = createBrowserRouter([
+    {
+        path : "/",
+        element : <AppLayout/>,
+        errorElement : <ErrorComponent/>,
+        children : [
+            {
```
Comment (vickydarlinn) [inline src/App.js:70]: You can add another child with index: true property to display some components by default in oulet.

File: src/App.js (hunk 1 above)
Comment (sow2245) [reply src/App.js:87]: I will doing it with next assignment which is covered in Clean Up Mess Class....Thank you so much for valuable comments

File: src/App.js (hunk 2 above)
Comment (sow2245) [reply src/App.js:70]: With Assignment 8 I will push that code

**Pull request merged**